extensionsGallery-insider.json

```diff
@@ -132,14 +132,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.1.5",
-							"lastUpdated": "2/27/2019",
+							"version": "0.2.0",
+							"lastUpdated": "10/2/2019",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/ServerReports/server-report-0.1.5.vsix"
```

Comment:
Were you planning on updating the whoIsActive extension as well? That was changed to not use sqlops.